config/jobs/common/dependabot.yaml

```diff
@@ -0,0 +1,42 @@
+periodics:
+- name: ci-gardener-dependabot-ok-to-test
```

Comment:
Instead of using an additional labeling job, you could also add `dependabot` to `plugins.trigger.trusted_apps`: https://github.com/kubernetes/test-infra/blob/9f82a909a9f806681e19febb3b305b7b66c668ed/prow/plugins/plugin-config-documented.yaml#L602-L606

Reply:
awesome 🚀